src/internal/kopia/wrapper_test.go

```diff
@@ -26,7 +26,7 @@ import (
 	"github.com/alcionai/corso/src/internal/data"
 	dataMock "github.com/alcionai/corso/src/internal/data/mock"
 	"github.com/alcionai/corso/src/internal/m365/collection/drive/metadata"
-	m365Mock "github.com/alcionai/corso/src/internal/m365/mock"
+	mockColl "github.com/alcionai/corso/src/internal/m365/mock/collection"
```

Comment:
nit: I would have stuck with the curent alias.  It doesn't match the package name as well, but `m365Mock.BackupCollection` is easier to grok than `mockColl.BackupCollection` (especially if/when we need to handle different providers).